app/scripts/modules/core/chaosMonkey/chaosMonkeyExceptions.component.ts

```diff
@@ -1,5 +1,6 @@
 import {module} from 'angular';
```

Comment:
would be good to cast `accountService` as the `AccountService` class in this file